invenio_requests/assets/semantic-ui/js/invenio_requests/components/FormattedInputEditor.js

```diff
@@ -5,77 +5,67 @@
 // under the terms of the MIT License; see LICENSE file for more details.
 import React, { Component } from "react";
 import Overridable from "react-overridable";
-import CKEditor from "@ckeditor/ckeditor5-react";
-import ClassicEditor from "@ckeditor/ckeditor5-build-classic";
+import { Editor } from "@tinymce/tinymce-react";
+import "tinymce/tinymce";
+import "tinymce/models/dom/model";
+import "tinymce/themes/silver";
+import "tinymce/icons/default";
+import "tinymce/plugins/table";
+import "tinymce/plugins/autoresize";
+import "tinymce/plugins/codesample";
+import "tinymce/plugins/link";
+import "tinymce/plugins/lists";
 import PropTypes from "prop-types";
 
-function MinHeightPlugin(editor) {
-  this.editor = editor;
-}
-
-function setMinHeight(minHeight) {
-  MinHeightPlugin.prototype.init = function () {
-    this.editor.ui.view.editable.extendTemplate({
-      attributes: {
-        style: {
-          minHeight: minHeight,
-        },
-      },
-    });
+class FormattedInputEditor extends Component {
+  handleBlur = (event, editor) => {
+    const modifiedContent = this.getModifiedContent(editor.getContent());
+    this.props.onBlur(event, modifiedContent);
   };
-  ClassicEditor.builtinPlugins.push(MinHeightPlugin);
-}
 
-const defaultConfig = {
-  toolbar: [
-    "heading",
-    "|",
-    "bold",
-    "italic",
-    "link",
-    "bulletedList",
-    "numberedList",
-    "Indent",
-    "Outdent",
-    "blockQuote",
-    "Undo",
-    "Redo",
-  ],
-};
+  handleChange = (event, editor) => {
+    const modifiedContent = this.getModifiedContent(editor.getContent());
+    this.props.onChange(event, modifiedContent);
+  };
 
-class FormattedInputEditor extends Component {
-  constructor(props) {
-    super(props);
-    const { minHeight } = this.props;
-    if (minHeight !== undefined) {
-      setMinHeight(minHeight);
-    }
+  getModifiedContent(content) {
+    const modifiedContent = `<div class="mce-content">${content}</div>`;
```

Comment:
could you add a comment in the code to explain why we wrap the content like this?
also let's wrap it in a bit more react way: 
```suggestion
    const modifiedContent = <div class="mce-content">{content}</div>;
```

Reply:
this doesn't work as tinyMCE getContent() returns a string